apps/innbyggers-flate/package.json

```diff
@@ -7,8 +7,8 @@
     "start": "VITE_ENDPOINT_HANDLER='MOCK' VITE_MOCK_SERVICE_RUNNER_PATH='/arbeidsmarkedstiltak/mockServiceWorker.js' vite --port 3005",
     "build:demo": "VITE_ENDPOINT_HANDLER='MOCK' vite build --base /",
     "build:pull-request": "VITE_ENDPOINT_HANDLER=DEV VITE_PR_ENV=pull_request vite build",
-    "build:dev": "VITE_FARO_URL=https://telemetry.ekstern.dev.nav.no/collect vite build",
-    "build": "VITE_FARO_URL=https://telemetry.nav.no/collect vite build",
+    "build:dev": "VITE_FARO_URL=https://telemetry.ekstern.dev.nav.no/collect vite build --mode staging",
+    "build": "VITE_FARO_URL=https://telemetry.nav.no/collect vite build --mode production",
```

Comment:
Jeg er kanskje alene om dette, men synes det er så hjernekapasistet med -mode, for env er prod for alle build, med mindre man setter det, staging er et ord vi ikke bruker for dev villøet, development er lockal, og ja 😅
er det mulig å gjøre det veldig tydlig navnene og hva som er forksjell på mode og env? Ev kan variablene in nher istedet? blir jo mye er og kanskje ikke så lett, men enten må alle variabler ut eller ingen, så det blir likt?
Kan man ev lage den i koden? som noen andre url-er? eller burde de andre url-ene ut i varibaler også?

Reply:
Enig i at det bør være likt hvertfall :) Jeg tror ikke vi får gjort noe med navnene siden de bestemmes av vite :/ Det kan hende det er fordi jeg er vant til backend-apper der ting er litt mer strukturert, men jeg synes det er veldig vanskelig å finne miljøspesifikk informasjon når den ligger hardkodet ulike steder i appen. For meg er det mye ryddigere å ha miljøspesifikk info i egne filer. Men det kan jo være fordi det er det jeg er vant til :) 

Jeg kan godt ta det bort og heller hardkode det sånn det var gjort med dialog-urlen, så kan heller dere som har mer greie på frontend finne ut hvordan dere vil gjøre det senere :)

Reply:
endret nå, så kan det heller tas som en egen oppgave hvis man ønsker å endre oppsettet :)